pages/services/kubernetes/1.0.0-1.9.3/index.md

```diff
@@ -2,7 +2,7 @@
 layout: layout.pug
 navigationTitle: Kubernetes 1.0.0-1.9.3
 title: Kubernetes 1.0.0-1.9.3
-menuWeight: 10
+menuWeight: 20
```

Comment:
Curious why the bump in menuWeight for the old version? New version is set to `10`

Reply:
This is to ensure strict ordering of versions in version grid - see https://docs.mesosphere.com/services/beta-kubernetes/ for the same output of the beta releases. This weight for old versions is bumped on import of new version to push the new version to the top of the version stack.

Reply:
What the great @jimmidyson said! Thanks, Jimmi.

Reply:
Thanks for the explanation @jimmidyson!